test/gemm_split_k/test_gemm_splitk_util.hpp

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2023, Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
This file and each new file: Shouldn't the licenses be for 2023? As far is it new file.